reactnative/RNPlayground/app/screens/AboutScreen.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FC } from "react"
```

Comment:
what is FC?

Reply:
`FC` means `Functional Component` and is a common interface provided by React as one of the two ways to type components - with `JSX.Element` being the other.

Each having pro's and con's, I found multiple blog posts similar to these. 
https://dev.to/xenoxdev/usage-of-reactfc-from-my-experience-22n7
https://dev.to/benhurmartins/typing-react-components-reactfc-or-jsxelement-p31

With this being a small project starting up I opted to stick with `FC` for now, happy to take input.

**NOTE: ** The issue with `FC` always implying children (often listed as a con) seems to only apply when project runs lower than React 18.